src/transactions/TransactionFrame.cpp

```diff
@@ -1731,12 +1737,16 @@ TransactionFrame::applyOperations(SignatureChecker& signatureChecker,
             app.getMetrics().NewTimer({"ledger", "operation", "apply"});
 
         uint64_t opNum{0};
+        OperationMetaArray opMetas(getNumOperations());
+        xdr::xvector<DiagnosticEvent> txDiagnostics;
```

Comment:
`txDiagnostics` should be moved down to where it's used.

Reply:
fixed